firmware/quadruna/BMS/src/app/states/app_allStates.c

```diff
@@ -13,6 +13,16 @@
 // Num of cycles for voltage and cell temperature values to settle
 #define NUM_CYCLES_TO_SETTLE (30U)
 
+uint32_t owcCounter = 0;
```

Comment:
Variables typically in snake_case
```suggestion
uint32_t iso_spi_state_counter = 0;
```